internal/pkg/task/errors.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
-	errNoSubnetFound = errors.New("no subnets found")
+	ErrNoSubnetFound = errors.New("no subnets found")
```

Comment:
Can we add a top-level comment here since we now have exported errors
```suggestion
// Errors returned while trying to run a task.
var (
	ErrNoSubnetFound = errors.New("no subnets found")
```